modules/performanceplatform/manifests/proxy_vhost.pp

```diff
@@ -53,6 +53,13 @@
 
   $graphite_servername = regsubst($servername, '\.', '_', 'G')
 
+  if ($serveraliases != undef) {
+    validate_array($serveraliases)
+    $vhost_names = $serveraliases
+  } else {
+    $vhost_names = [$servername]
```

Comment:
Ah, maybe the fact that the `$serveraliases` parameter isn't introduced with this change wasn't apparent?

I've force-pushed with an updated commit comment.